src/connect.js

```diff
@@ -129,10 +129,10 @@ export default (mapFirebaseToProps = defaultMapFirebaseToProps) => {
       }
 
       render() {
+        const subscriptionProps = this.state.subscriptionsState
         const firebaseProps = mapFirebase(this.props, this.ref, this.firebaseApp)
         const actionProps = pickBy(firebaseProps, isFunction)
-        const subscriptionProps = this.state.subscriptionsState
-        const props = mergeProps(actionProps, subscriptionProps, this.props)
+        const props = mergeProps(this.props, subscriptionProps, actionProps)
```

Comment:
Why can't we do
```js
const props = mergeProps(this.props, { ...actionProps, ...subscriptionProps })
```
instead? Why should the developer get three arguments to `mapProps`?